agendas/2022/2022-01-06.md

```diff
@@ -0,0 +1,88 @@
+        That means we expect most who join the meeting to participate in
+      discussion. If you'd rather just watch, check out our YouTube[1].
+
+      - Please include the organization (or project) you represent, and the
+        location (including country code[2]) you expect to be located in during
+        the meeting.
+
+      - If you're willing to help take notes, add "✏️" after your name
+        (eg. Ada Lovelace ✏). This is hugely helpful!
+
+   2. If relevant, add your topic to the agenda (sorted by expected time).
+
+      - Every agenda item has four parts: 1) the topic, 2) an expected time
+        constraint, 3) who's leading the discussion, and 4) a list of any
+        relevant links (RFC docs, issues, PRs, presentations, etc). Follow the
+        format of existing agenda items.
+
+   3. Review our guidelines and agree to our Spec Membership & CLA.
+
+      - Review and understand our Spec Membership Agreement, Participation
+        Guidelines and Code of Conduct. You'll find links to these in the
+        first agenda item of every meeting.
+
+      - If this is your first time, our bot will comment on your Pull Request
+        with a link to our Spec Membership & CLA. Please follow along and agree
+        before your PR is merged.
+
+        Your organization may sign this for all of its members. To set this up,
+        please ask operations@graphql.org.
+
+PLEASE TAKE NOTE:
+
+  - By joining this meeting you must agree to the Specification Membership
+    Agreement, Participation Guidelines, and Code of Conduct.
+
+  - Meetings are recorded and made available on YouTube[1], by joining you
+    consent to being recoded.
+
+[1] Youtube: https://www.youtube.com/channel/UCERcwLeheOXp_u61jEXxHMA
+[2] Country codes: https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes#Current_ISO_3166_country_codes
+
+-->
+
+# GraphQL WG – January 2022
+
+The GraphQL Working Group meets monthly to discuss proposed additions to the
+[GraphQL Specification](https://github.com/graphql/graphql-spec) and other
+relevant topics to core GraphQL projects. This is an open meeting in which
+anyone in the GraphQL community may attend.
+
+*To attend this meeting or add agenda, send a Pull Request against this file.*
+
+- **Date & Time**: [January 6th 2022 19:00 - 21:00 UTC](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2022&month=1&day=6&hour=19&min=0&sec=0&p1=224&p2=179&p3=136&p4=268&p5=367&p6=438&p7=240&iv=0), view the [calendar](https://calendar.google.com/calendar/embed?src=linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com), or subscribe ([Google Calendar](https://calendar.google.com/calendar?cid=bGludXhmb3VuZGF0aW9uLm9yZ19pazc5dDl1dWoycDMyaTNyMjAzZGd2NW1vOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t), [ical file](https://calendar.google.com/calendar/ical/linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com/public/basic.ics)).
```

Comment:
To make it easier to spot mistakes in the date/time in future, perhaps we can use reference-style links for the unchanging links:

```md
- **Date & Time**: [January 6th 2022 19:00 - 21:00 UTC](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2022&month=1&day=6&hour=19&min=0&sec=0&p1=224&p2=179&p3=136&p4=268&p5=367&p6=438&p7=240&iv=0), view the [calendar][], or subscribe ([Google Calendar][], [ical file][]).

[calendar]: https://calendar.google.com/calendar/embed?src=linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com
[Google Calendar]: https://calendar.google.com/calendar?cid=bGludXhmb3VuZGF0aW9uLm9yZ19pazc5dDl1dWoycDMyaTNyMjAzZGd2NW1vOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t
[ical file]: https://calendar.google.com/calendar/ical/linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com/public/basic.ics
```

> - **Date & Time**: [January 6th 2022 19:00 - 21:00 UTC](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2022&month=1&day=6&hour=19&min=0&sec=0&p1=224&p2=179&p3=136&p4=268&p5=367&p6=438&p7=240&iv=0), view the [calendar][], or subscribe ([Google Calendar][], [ical file][]).
>
> [calendar]: https://calendar.google.com/calendar/embed?src=linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com
> [Google Calendar]: https://calendar.google.com/calendar?cid=bGludXhmb3VuZGF0aW9uLm9yZ19pazc5dDl1dWoycDMyaTNyMjAzZGd2NW1vOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t
> [ical file]: https://calendar.google.com/calendar/ical/linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com/public/basic.ics